From f54df6d82b9ee4986d3746c8fe767b11d59a53eb Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Thu, 18 Feb 2016 13:21:59 +0100
Subject: [PATCH] man: ip-address: document mngtmpaddr and noprefixroute flags

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1231898
Upstream Status: iproute2.git commit f7afa999520d3

commit f7afa999520d3f93114c296e37e4ccea84d2c6de
Author: Phil Sutter <phil@nwl.cc>
Date:   Fri Sep 18 20:17:03 2015 +0200

    man: ip-address: document mngtmpaddr and noprefixroute flags

    Signed-off-by: Phil Sutter <phil@nwl.cc>
---
 man/man8/ip-address.8.in | 18 +++++++++++++++++-
 1 file changed, 17 insertions(+), 1 deletion(-)

diff --git a/man/man8/ip-address.8.in b/man/man8/ip-address.8.in
index 43611b9..159d906 100644
--- a/man/man8/ip-address.8.in
+++ b/man/man8/ip-address.8.in
@@ -67,7 +67,7 @@ temporary " | " CONFFLAG-LIST " ]"
 
 .ti -8
 .IR CONFFLAG " := "
-.RB "[ " home " | " nodad " ]"
+.RB "[ " home " | " mngtmpaddr " | " nodad " | " noprefixroute " ]"
 
 .ti -8
 .IR LIFETIME " := [ "
@@ -181,10 +181,26 @@ outgoing connections. Defaults to
 RFC 6275.
 
 .TP
+.B mngtmpaddr
+(IPv6 only) make the kernel manage temporary addresses created from this one as
+template on behalf of Privacy Extensions (RFC3041). For this to become active,
+the \fBuse_tempaddr\fP sysctl setting has to be set to a value greater than
+zero.  The given address needs to have a prefix length of 64. This flag allows
+to use privacy extensions in a manually configured network, just like if
+stateless auto-configuration was active.
+
+.TP
 .B nodad
 (IPv6 only) do not perform Duplicate Address Detection (RFC 4862) when
 adding this address.
 
+.TP
+.B noprefixroute
+Do not automatically create a route for the network prefix of the added
+address, and don't search for one to delete when removing the address. Changing
+an address to add this flag will remove the automatically added prefix route,
+changing it to remove this flag will create the prefix route automatically.
+
 .SS ip address delete - delete protocol address
 .B Arguments:
 coincide with the arguments of
-- 
1.8.3.1

